packages/core/src/lib/implementation/execute-plugin.unit-test.ts

```diff
@@ -176,18 +186,18 @@ describe('executePlugins', () => {
             outputTransform: (outputs: unknown): Promise<AuditOutputs> =>
               Promise.resolve([
                 {
-                  slug: (outputs as AuditOutputs)[0]!.slug,
-                  score: 0.3,
-                  value: 16,
-                  displayValue: '16.0.0',
+                  slug: (outputs as AuditOutputs)[0]?.slug || '',
```

Comment:
```suggestion
                  slug: (outputs as AuditOutputs)[0]!.slug,
```

Reply:
These are consequences of a `unit-test.ts` instead of `unit.test.ts` file name as pointed out [here](https://github.com/code-pushup/cli/pull/487#discussion_r1517510324).